nemo/collections/asr/beam_search_decoder.py

```diff
@@ -59,7 +59,9 @@ def output_ports(self):
         # return {"predictions": NeuralType(VoidType())}
         return {"predictions": NeuralType(('B', 'T'), PredictionsType())}
 
-    def __init__(self, vocab, beam_width, alpha, beta, lm_path, num_cpus, cutoff_prob=1.0, cutoff_top_n=40):
+    def __init__(
+        self, vocab, beam_width, alpha, beta, lm_path, num_cpus, cutoff_prob=1.0, cutoff_top_n=40, input_tensor=True
```

Comment:
Can you please add a line describing `input_tensor` parameter into the class docstring?

Reply:
done